src/test/java/teammates/test/cases/browsertests/InstructorFeedbackSessionsPageUiTest.java

```diff
@@ -586,12 +586,11 @@ private void testDeleteAction() throws Exception {
         String sessionName = "Long Instruction Test ##";
 
         // refresh page
-        feedbackPage = getFeedbackPageForInstructor(idOfInstructorWithSessions);
-        feedbackPage.clickAndCancel(feedbackPage.getDeleteLink(courseId, sessionName));
+        feedbackPage = getFeedbackPageForInstructor(idOfInstructorWithSessions).deleteSession(courseId, sessionName);
+        assertTrue(feedbackPage.getTextsForAllStatusMessagesToUser()
+                .contains(Const.StatusMessages.FEEDBACK_SESSION_MOVED_TO_RECYCLE_BIN));
         assertNotNull("session should not have been deleted",
                       BackDoor.getFeedbackSession(courseId, sessionName));
-
-        feedbackPage.clickAndConfirm(feedbackPage.getDeleteLink(courseId, sessionName));
         feedbackPage.verifyHtmlMainContent("/instructorFeedbackDeleteSuccessful.html");
```

Comment:
Hmm for soft delete you can also test cancelling the soft delete as well (since it is possible to cancel)

Reply:
Oh actually in `InstructorFeedbackSessionsPage` there won't be a JavaScript confirmation for soft-deleting a session. The confirmation comes out when deleting a session from `InstructorHomePage`. This follows the behaviors of archiving courses & soft-deleting courses.